neondatabase/autoscaling
# Add e2e tests

This PR adds e2e tests for autoscaling. Some scripts from `neonvm` are used for environment preparation, so I've added `neonvm` as a submodule (for the time being, until we properly merge these repos).

Test `tests/e2e/autoscaling`:
- Create VM (1cpu, 1Gi)
- Start workload
- Check VM upscaled (to max 4cpu 4Gi)
- Stop workload
- Check VM downscaled (back to 1cpu 1Gi)
